packages/notebook/test/widget.spec.ts

```diff
@@ -43,7 +43,8 @@ const notebookConfig = {
   observedTopMargin: '1000px',
   observedBottomMargin: '1000px',
   maxNumberOutputs: 50,
-  disableDocumentWideUndoRedo: true
+  disableDocumentWideUndoRedo: true,
+  renderingLayout: 'default' as 'default' | 'side-by-side'
```

Comment:
Maybe `'default' | 'side-by-side'` should be made a `type` for improved readability and maintainability?

Reply:
Done. Thx @jtpio